plugins/arDominionPlugin/css/less/scaffolding.less

```diff
@@ -1552,6 +1552,37 @@ body.login #content {
   }
 }
 
+// GDPR Privacy Message banner
+@media (max-width: 767px) {
+  #privacy-message {
+    margin-left: -20px;
+    margin-right: -20px;
+  }
+}
+
+#privacy-message > .alert {
+  margin-bottom: 0px;
+  margin-top: 0px;
+  text-align: center;
+  padding: 8px 35px 18px 14px !important;
```

Comment:
Is this `!important` really necessary, I'd try to find a better selector to see if we can avoid using it.

Reply:
Testing this again, it looks like '!important' is not necessary after all. Removed....